inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -1614,6 +1614,8 @@ cdef class BlobBuffer:
         cdef SizeVector shape
         if len(representation_shape) == 0:
             shape = desc.getDims()
+            if layout_int_to_str_map[desc.getLayout()] == 'SCALAR':
+                shape = [1]
```

Comment:
@akuporos  After this change `blob.buffer[:]` is 1-elem array. Do you see any negative consequences of it?

Reply:
No